src/grammar/grammar.ohm

```diff
@@ -193,6 +194,7 @@ Tact {
                     | null
                     | ExpressionInitOf
                     | ExpressionString
+                    | ExpressionUnary
```

Comment:
Let's refactor the grammatical category for unary expressions instead (see above)
```suggestion
```

It's a bit easier to reason about since it removes one level of indirection